corona-warn-app/cwa-app-android
# Certificate poster fragment (EXPOSUREAPP-9188, EXPOSUREAPP-9189, EXPOSUREAPP-9190)

Certificate (Vaccination, Recovery and Test) poster fragment.
Actual PDF poster creation would be handled in another PR.

How to test:
For each certificate type:
1) scan certificate
2) navigate to certificate details
3) three-dots-menu -> Print
4) on Info screen press continue button
5) poster fragment should appear
6) check in logs "Certificate found: " with certificate name owner (this log is for test purpose only - would be deleted later).

TODO:
- [x] Toolbar back button mystery (back arrow does not show right now)
- [x] Check if we need to replace layout with new one
